client/lib/importer/test/api-interaction.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import Dispatcher from 'dispatcher';
+import nock from 'nock';
+import partial from 'lodash/function/partial';
+
+import { fetchState } from '../actions';
+import { actionTypes } from '../constants';
+import store from '../store';
+
+const fetchTestState = partial( fetchState, 0 );
```

Comment:
This is something I didn't think about because it didn't affect the test I was conducting. I have changed this in 843e62c89f29830477508d8b80469cb53d3e39da to use `en.blog.wordpress.com` as the test site, but since it's still being mocked with `nock` there should still be no actual network call.